common/middleware/set-primary-navigation.js

```diff
@@ -0,0 +1,48 @@
+const { format } = require('date-fns')
+const { get } = require('lodash')
+
+const permissions = require('./permissions')
+
+const baseRegex =
```

Comment:
Was going to suggest deferring this to after the running of this middleware but I see that's not possible for all the routes because they have a different structure. Ideally that's what you'd do and just let the relevant controllers mark that a section is active. The form-wizard derived stuff could call such middleware in the locals phase (ie. just before rendering) but the other route controllers would need to do so before calling this.render

Reply:
Yep, it did feel a bit smelly when writing it 😞 

But I couldn't see another way to effectively highlight the nav items correctly.

Reply:
It's also made more complicated by our current redirect on `/moves`. But as @barbaracassani and I have discussed this was always more of a temporary thing that once we start actually introducing dashboards at those routes, the redirects can go.

But it was a case of priority in MVP days that we focusing on building the create and list journeys first over the summary dashboards.